assets.js

```diff
@@ -0,0 +1,14 @@
+//const CSS = [
+
+//];
+const JS = [
+    'requirejs/require.js',
+    'alameda/alameda.js',
+    'webcomponents.js-2/webcomponents-lite.min.js',
+    'document-register-element/build/document-register-element.js',
+    'native-promise-only/lib/npo.src.js',
+    'resize-observer-polyfill/dist/ResizeObserver.js',
+    'whatwg-fetch/fetch.js'
```

Comment:
I think alameda and requirejs are the only two that need this special method. The rest can just use the bundle method that @thornbill started using.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/549/commits/651a53d64a2d44abaa7bfd9bf1bf58e5a7fe4122